pkg/loop/internal/core/services/capability/capabilities_registry.go

```diff
@@ -21,6 +23,44 @@ type capabilitiesRegistryClient struct {
 	grpc pb.CapabilitiesRegistryClient
 }
 
+func toDON(don *pb.DON) capabilities.DON {
```

Comment:
nit: Maybe call it "toCapDON"?

Reply:
I'm using it for the workflow DON struct too